recipes/pyworkout-toolkit/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pyworkout-toolkit" %}
+{% set version = "0.0.21" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/triskadecaepyon/pyworkout-toolkit/archive/v{{ version }}.tar.gz
+  sha256: eb188622fac6a4585c7122fcf4b0c1d12f879e00b084b15a9b435d420c9b4960
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python
```

Comment:
Looks like the package is Python 3 only based on the metadata here:

https://github.com/triskadecaepyon/pyworkout-toolkit/blob/master/setup.py#L40

If that is correct a `noarch` package will make it installable on Python 2 and that would be a broken package. You can still do a `noarch` package but you would need to constrain here and below to `python >=3`.